2004-02-23  Elena Zannoni  <ezannoni@redhat.com>

        * gdb.gdb/selftest.exp: Make sure that the debug directory is
        set up properly.
        * gdb.gdb/complaints.exp: Ditto.
        * gdb.gdb/xfullpath.exp: Ditto.
        * gdb.gdb/observer.exp: Ditto.

Index: gdb-6.8.50.20090226/gdb/testsuite/gdb.gdb/complaints.exp
===================================================================
--- gdb-6.8.50.20090226.orig/gdb/testsuite/gdb.gdb/complaints.exp	2009-01-03 06:58:04.000000000 +0100
+++ gdb-6.8.50.20090226/gdb/testsuite/gdb.gdb/complaints.exp	2009-02-26 22:10:48.000000000 +0100
@@ -302,13 +302,13 @@ proc find_gdb { arg } {
 set GDB_FULLPATH [find_gdb $GDB]
 
 # Remove any old copy lying around.
-remote_file host delete x$tool
+#remote_file host delete x$tool
 
 gdb_start
 
-set file [remote_download host $GDB_FULLPATH x$tool]
+#set file [remote_download host $GDB_FULLPATH x$tool]
 
-set setup_result [setup_test $file ]
+set setup_result [setup_test $GDB_FULLPATH ]
 if {$setup_result <0} then {
     return -1
 }
@@ -319,4 +319,4 @@ test_short_complaints
 test_empty_complaints
 
 gdb_exit;
-catch "remote_file host delete $file";
+#catch "remote_file host delete $file";
Index: gdb-6.8.50.20090226/gdb/testsuite/gdb.gdb/observer.exp
===================================================================
--- gdb-6.8.50.20090226.orig/gdb/testsuite/gdb.gdb/observer.exp	2009-02-17 20:52:27.000000000 +0100
+++ gdb-6.8.50.20090226/gdb/testsuite/gdb.gdb/observer.exp	2009-02-26 22:11:40.000000000 +0100
@@ -258,13 +258,13 @@ proc find_gdb { arg } {
 set GDB_FULLPATH [find_gdb $GDB]
 
 # Remove any old copy lying around.
-remote_file host delete x$tool
+#remote_file host delete x$tool
 
 gdb_start
-set file [remote_download host $GDB_FULLPATH x$tool]
-set result [test_observer $file];
+#set file [remote_download host $GDB_FULLPATH x$tool]
+set result [test_observer $GDB_FULLPATH];
 gdb_exit;
-catch "remote_file host delete $file";
+#catch "remote_file host delete $file";
 
 if {$result <0} then {
     warning "Couldn't test self"
Index: gdb-6.8.50.20090226/gdb/testsuite/gdb.gdb/selftest.exp
===================================================================
--- gdb-6.8.50.20090226.orig/gdb/testsuite/gdb.gdb/selftest.exp	2009-02-26 22:09:59.000000000 +0100
+++ gdb-6.8.50.20090226/gdb/testsuite/gdb.gdb/selftest.exp	2009-02-26 22:10:48.000000000 +0100
@@ -551,13 +551,13 @@ proc find_gdb { arg } {
 set GDB_FULLPATH [find_gdb $GDB]
 
 # Remove any old copy lying around.
-remote_file host delete x$tool
+#remote_file host delete x$tool
 
 gdb_start
-set file [remote_download host $GDB_FULLPATH x$tool]
-set result [test_with_self $file];
+#set file [remote_download host $GDB_FULLPATH x$tool]
+set result [test_with_self $GDB_FULLPATH];
 gdb_exit;
-catch "remote_file host delete $file";
+#catch "remote_file host delete $file";
 
 if {$result <0} then {
     warning "Couldn't test self"
Index: gdb-6.8.50.20090226/gdb/testsuite/gdb.gdb/xfullpath.exp
===================================================================
--- gdb-6.8.50.20090226.orig/gdb/testsuite/gdb.gdb/xfullpath.exp	2009-01-03 06:58:04.000000000 +0100
+++ gdb-6.8.50.20090226/gdb/testsuite/gdb.gdb/xfullpath.exp	2009-02-26 22:10:48.000000000 +0100
@@ -179,13 +179,13 @@ proc find_gdb { arg } {
 set GDB_FULLPATH [find_gdb $GDB]
 
 # Remove any old copy lying around.
-remote_file host delete x$tool
+#remote_file host delete x$tool
 
 gdb_start
-set file [remote_download host $GDB_FULLPATH x$tool]
-set result [test_with_self $file];
+#set file [remote_download host $GDB_FULLPATH x$tool]
+set result [test_with_self $GDB_FULLPATH];
 gdb_exit;
-catch "remote_file host delete $file";
+#catch "remote_file host delete $file";
 
 if {$result <0} then {
     warning "Couldn't test self"
